docs/plugins/asg_rollout.md

```diff
@@ -0,0 +1,85 @@
+# Auto-Scaling Group LaunchConfig Rollout Plugin
+
+## Overview
+
+This plugin adds support for rolling out changes to Auto Scaling
+Groups that have happened after a stack update. It supports various
+pre- and post-actions on the instances (using Moonshot's native SSH
+support).
+
+## Example
+
+The ASGRollout class is intended to be used within a method in your
```

Comment:
For this reason, I don't really like the idea considering this class a plugin, and it being in the `Plugins` module. Correct me if I'm wrong, but it won't work like a plugin in it's current sense.